docs/crio.8.md

```diff
@@ -148,7 +166,11 @@ If `hooks_dir` is unset, CRI-O will currently default to `/usr/share/containers/
 
 **--storage-opt**: OCI storage driver option (no default)
 
-**--uid-mappings**: Specify the UID mappings to use for user namespace.
+**--stream-address**="": bind address for streaming socket
+
+**--stream-port**="":  bind port for streaming socket (default: "0")
+
+**--uid-mappings**: Specify the UID mappings to use for user namespace
 
 **--version, -v**: Print the version
```

Comment:
Do we have any examples anywhere for these?

Reply:
Examples of the flag being used?

Reply:
There are no examples on this page, it would be nice to have some.  Not a requirement for this PR getting through though.